server/index.js

```diff
@@ -29,14 +29,10 @@ app.get('/api/songs/:songId', (req, res) => {
   });
 });
 
-
 app.listen(port, () => console.log(`Your server has connected and is listening on port: ${port}!!`));
 
-  // used to populate the db
+// used to populate the db
 // app.get('/api/dbSave', (req, res) => {
-//   for (let i = 0; i < data.songs.length; i++) {
-//     db.save(null, data.songs[i]);
-//     console.log(data.songs[i])
-//   }  
-//   res.send('Great Success!!!')
-// })
```

Comment:
Why are you deleting the for loop? or what exactly are you doing here?

Reply:
I had issues, with reimplementing modified sample data, my usual go-to is simplifying things, thus deleting the for loop. I eventually figured it out. I had modified my schema to use _id instead of id and the objects I was passing in did not have that modification.